po/POTFILES.in

```diff
@@ -180,6 +180,7 @@ config/services/nrpe.xml
 config/services/ntp.xml
 config/services/nut.xml
 config/services/openvpn.xml
+config/services/otlp.xml
```

Comment:
IANA uses the name `opentelemetry`. We should probably use the same.

https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=4317

And OpenTelementry says the short version is "OTel". We should probably use the full name though.

https://opentelemetry.io/docs/

Reply:
I went with the name of the protocol, but I can switch it to opentelemtry easily enough.

Reply:
I think we should use the IANA name, `opentelemetry`.

Reply:
I'll switch to that, should I leave the `<short>OTLP</short>` or switch that to `OTeL`?